src/app/api/dict/generate/description/route.ts

```diff
@@ -7,7 +7,9 @@ import { revalidatePath } from "next/cache";
 export async function POST(req: Request) {
   try {
     console.log("### Start generating description for word");
+    console.time("Connect DB");
```

Comment:
This will incur some error although harmless. The timer of the same label string can only be created once, meaning when a second request is creating a timer while the first request is still trying to connect to the DB, the timer creation will fail. I suggest adding timing system later when scalability becomes a priority to this project